packages/miniapp-utils/src/helpers/analytics/types.ts

```diff
@@ -0,0 +1,24 @@
+import type { AnalyticsInstance, AnalyticsPlugin } from 'analytics'
+
+export type AnalyticsConfig = {
+    app?: string
+    version?: string | number
```

Comment:
what is version used for? is it the version of the app?

Reply:
Yep, that's apps version:
https://getanalytics.io/api/#configuration